src/components/text/docs.mdx

```diff
@@ -0,0 +1,29 @@
+---
+componentName: Text
+---
+
+The `Text` component should be used for easily styling `p` elements to conform to the design system.
+
+## Props
+
+In addition to the props described in the table below, this component also accepts any other valid props for heading elements. A caveat:
```

Comment:
```suggestion
In addition to the props described in the table below, this component also accepts any other valid props for its `asElement`. A caveat:
```

Reply:
Fixed this along with the other issue in this file in https://github.com/hashicorp/dev-portal/pull/88/commits/443f3604c283e3b7f54d68f952e9ce00b959507b. 😊